### PR TITLE
Explicit "Add Parameter" Button in Query Editor

### PR DESCRIPTION
**Pull request by kravets-levko**
Implemented as described in getredash/redash#2644

Screenshots:

image
image
image

### CODE REVIEW

**Comment by kravets-levko** [comment]
@kocsmy yes, you're right, there are so much screenshots in issue discussion :smile: Fixed

File: client/app/services/query.js [hunk 1]
```diff
@@ -129,6 +129,14 @@ class Parameters {
     return this.query.options.parameters;
   }
 
+  add(parameterDef) {
+    if (isObject(parameterDef)) {
```
Comment (arikfr) [inline client/app/services/query.js]: When is it not an object?

Comment (kravets-levko) [reply client/app/services/query.js]: It's just for a case

Comment (arikfr) [reply client/app/services/query.js]: What case? :)

**Comment by kravets-levko** [comment]
@arikfr Updated PR, please review it again. Thanks!

**Comment by arikfr** [comment]
🍷

**Pull request merged**